f2/src/components/formik/datePicker/index.js

```diff
@@ -79,8 +79,8 @@ const DateEntry = ({ field, form, ...props }) => {
       <Form.Group controlId={props.id}>
         <DateLabel>{props.label}</DateLabel>
         <DateInput
-          hasError={error}
-          dateType={dateType}
+          haserror={error}
+          datetype={dateType}
           type="text"
           {...field}
           maxLength={length}
```

Comment:
If we need to change all properties to lower cases, how about "maxLength" here?

Reply:
maxLength is a valid DOM property. The problem here (I should have mentioned in the description) is that Styled-System and Emotion are causing all props to be passed to the rendered element. 

React syntax dictates that DOM properties need to be declared in camel case. To include custom properties they need to be written in lower case.

Reply:
https://reactjs.org/docs/introducing-jsx.html#specifying-attributes-with-jsx

Reply:
That makes sense. Thanks, Justin!